subsys/spm/Kconfig

```diff
@@ -55,6 +55,14 @@ config SPM_BOOT_SILENTLY
 endif #IS_SPM
 
 if IS_SPM || SPM
+
+config SPM_NS_DEBUG_LOGS_DUMP
```

Comment:
IMO there should be a warning from CMake if this is set. We might want to have a warning in kconfig as well. This is typically done for all options which must be disabled for production (debug keys, verbose assert etc).

Reply:
```
if SPM_NS_DEBUG_LOGS_DUMP:
    comment "SPM_NS_DEBUG_LOGS_DUMP can only be used for debugging"
```